proto/nft_transfer/v1/packet.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package chainmain.nft_transfer.v1;
+
+option go_package = "github.com/crypto-org-chain/chain-main/x/nft-transfer/types";
+
+// NonFungibleTokenPacketData defines a struct for the packet payload
+// See NonFungibleTokenPacketData spec:
+// https://github.com/cosmos/ibc/tree/master/spec/app/ics-721-nft-transfer#data-structures
+message NonFungibleTokenPacketData {
+  // the class_id of tokens to be transferred
+  string class_id = 1;
+  // the class_uri of tokens to be transferred
+  string class_uri = 2;
```

Comment:
token_ids count is equal to token_uris?
some comment would be good

Reply:
Done